app/controllers/shopping_list_controller.rb

```diff
@@ -1,6 +1,5 @@
 class ShoppingListController < ApplicationController
   def index
-    # @items = RecipeFood.where(recipe_id: params[:id])
     @items = RecipeFood.includes(:food, :new_recipe)
       .select('recipe_foods.*, foods.price * recipe_foods.quantity AS total_price')
       .joins(:food)
```

Comment:
![image](https://github.com/YoutLone/Recipe/assets/93039359/174e3ce5-e4a0-4ffa-b9e4-26214d7f9cf0)

- [ ] Great work on adding the foods to the recipe but I guess there is a problem that came out after this fix :thinking: because i have added 10 salts in the foods and then i have added 15 salts in my recipe so the shopping list should count the missing food means 5 salts because i already had 10 salts so kindly fix this issue to make your app work as expected :+1: